internal/metrics/querycoord.go

```diff
@@ -0,0 +1,150 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/milvus-io/milvus/internal/util/typeutil"
+)
+
+const (
+	QueryCoordMetricLabelSuccess = "success"
+	QueryCoordMetricLabelFail    = "fail"
+	QueryCoordMetricLabelTotal   = "total"
+)
+
+var queryCoordLoadBuckets = []float64{0, 10, 100, 1000, 10000, 100000, 1000000, 10000000}
+
+var (
+	QueryCoordNumCollections = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.QueryCoordRole,
+			Name:      "num_collections",
+			Help:      "Number of collections in QueryCoord.",
+		}, []string{
+			"num_collections",
+		})
+
+	QueryCoordNumEntities = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: milvusNamespace,
+			Subsystem: typeutil.QueryCoordRole,
+			Name:      "num_entities",
+			Help:      "Number of entities in collection.",
+		}, []string{
+			"num_entities",
```

Comment:
add collection_id label to distinguish different collections.

Reply:
fixed